index.html

```diff
@@ -20,27 +22,160 @@
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card-3">
+        <div class="card-content"></div>
+        <img src="./images/tonic2.png" alt="Tonic" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
 
-    <section class="content">
-      <h1 class="title">
-        I'm Luis <br>Glad to see you!
-      </h1>
-      <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-      <h2 class="connect"> <a href="#">Let's connect</a></h2>
+      <article class="card-4">
+        <div class="card-content"></div>
+        <img src="./images/multipost2.png" alt="Multi-Post Project" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Project</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
```

Comment:
- [ ] As preciously requested, implement this feature for technology stacks listed. Do this for all your projects
> Kindly replace `div` with `ul` and `li` tags to list the technologies.

```
        <ul>
          <li>html</li>
          <li>Javascript</li>
          <li>CSS</li>
        </ul>
```

Reply:
Just saw where I missed this in the previous PR. Fixed now.